pkgs/development/guile-modules/guile-git/default.nix

```
@@ -0,0 +1,53 @@
+{ stdenv, fetchFromGitHub, fetchFromGitLab, guile, libgit2
+, autoreconfHook, pkg-config, texinfo
+}:
+
+let
+  bytestructures = stdenv.mkDerivation rec {
+    pname = "scheme-bytestructures";
+    version = "1.0.7";
+
+    src = fetchFromGitHub {
+      owner = "TaylanUB";
+      repo = pname;
+      rev = "v${version}";
+      sha256 = "0q0habjiy3h9cigb7q1br9kz6z212dn2ab31f6dgd3rrmsfn5rvb";
+    };
+
+    postConfigure = ''
+      sed -i '/moddir\s*=/s%=.*%=''${out}/share/guile/site%' Makefile;
+      sed -i '/godir\s*=/s%=.*%=''${out}/share/guile/ccache%' Makefile;
+    '';
+
+    nativeBuildInputs = [ autoreconfHook pkg-config ];
+    buildInputs = [ guile ];
+  };
+in stdenv.mkDerivation rec {
+  pname = "guile-git";
+  version = "0.3.0";
+
+  src = fetchFromGitLab {
+    owner = pname;
```

```suggestion
    owner = "guile-git";
```

The owner happens to match `pname`, but the two values don't change together. Preferably `repo` should be explicit as well.